Moto/Z2PLay/res/values/config.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+	<bool name="config_enableAutoPowerModes">true</bool>
+	<integer name="config_autoPowerModeAnyMotionSensor">30</integer>
+	<bool name="config_wifi_dual_band_support">true</bool>
+	<bool name="config_wifi_fast_bss_transition_enabled">true</bool>
+	<bool name="config_wifi_background_scan_support">true</bool>
+	<bool name="config_wifi_enable_disconnection_debounce">true</bool>
+	<bool name="config_setColorTransformAccelerated">true</bool>
+	<bool name="config_nightDisplayAvailable">false</bool>
+	<bool name="config_suspendWhenScreenOffDueToProximity">true</bool>
+	<bool name="config_hotswapCapable">true</bool>
+	<bool name="config_bluetooth_le_peripheral_mode_supported">true</bool>
+	<integer name="config_bluetooth_operating_voltage_mv">3300</integer>
+	<bool name="skip_restoring_network_selection">true</bool>
+	<bool name="config_showNavigationBar">true</bool>
+	<string name="config_dozeComponent">com.android.systemui/com.android.systemui.doze.DozeService</string>
+	<bool name="config_dozeAfterScreenOff">true</bool>
+	<bool name="config_powerDecoupleInteractiveModeFromDisplay">true</bool>
+	<bool name="config_cellBroadcastAppLinks">true</bool>
+	<integer name="config_mobile_mtu">1410</integer>
+	<bool name="config_speed_up_audio_on_mt_calls">true</bool>
+	<bool name="config_auto_attach_data_on_creation">false</bool>
+	   <string name="config_wifi_tcp_buffers">131072,262144,3145728,4096,221184,3145728</string>
+    <bool name="config_unplugTurnsOnScreen">true</bool>
+    <bool name="config_lidControlsSleep">false</bool>
+    <bool name="config_useDevInputEventForAudioJack">false</bool>
```

Comment:
This sounds wrong, Qualcomm device have that to true
Where did you get this value from?